api/app/src/main/kotlin/packit/model/dto/CreateRole.kt

```diff
@@ -1,9 +1,14 @@
 package packit.model.dto
 
+import jakarta.validation.constraints.Pattern
 import org.jetbrains.annotations.NotNull
 
 data class CreateRole(
     @field:NotNull
+    @field:Pattern(
+        regexp = "^[a-zA-Z0-9]+(\\s[a-zA-Z0-9]+)*\$",
```

Comment:
same regex as FE

Reply:
The `\$` at the end seems suspicious. It should just be `$` I think.

You could use a character class (eg. `\w` or `\p{Alnum}`) instead of `[a-zA-Z0-9]`, though that’s just a matter of taste.

Finally I reckon using a literal space rather than `\s` may be better to avoid tabs and newlines and whatnot

https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html?is-external=true